### PR TITLE
Solved issue with the reveal action not converting footsteps to brainfacts

### PR DESCRIPTION
**Pull request by Vilmavds**
### Changes
- Solved issue #63
- The `reveal()` call was commented out in the `GameController,` so I basically uncommented it. 
- After that I also simplified the logic of the `reveal()` method to take one argument instead of four ones. This simplified both the logic within the function and the calling of the function. 
- Updated the ActionControllerTests
### Reviewer
- Added @Joakim-Johansson since he is also working with the reveal

### CODE REVIEW

File: Project/core/src/main/java/View/screen/GameScreenComponents/RevealButton.java [hunk 1]
```diff
@@ -27,10 +25,7 @@ public RevealButton(GameController gameController, Skin skin) {
         this.addListener(new ChangeListener() {
             @Override
             public void changed(ChangeEvent event, Actor actor) {
-                // Not sure if this is working now
-                Board board = gameController.getGame().getBoard();
-                int[] playerPosition = board.getPlayerCoord(player);
-                NormalCell cell = (NormalCell) board.getCell(playerPosition[0], playerPosition[1]);
+                AbstractCell cell = gameController.getGame().getCurrentPlayerCell();
```
Comment (Joakim-Johansson) [inline Project/core/src/main/java/View/screen/GameScreenComponents/RevealButton.java:28]: This validation could probably be done in the actionHandler as actionHandler already retrieves the current cell which would remove some redundancy. I think`MOVE` does it similarly, where the handler is always run but it just runs `break` if it's invalid.

**Review by Joakim-Johansson — APPROVED** [review]
Really like how you updated the `reveal` function, made it much smoother. Only had a small suggestion which you can see in the comments, otherwise it looks solid. This probably won't have any trouble merging with my issues either, which is nice.

File: Project/core/src/main/java/View/screen/GameScreenComponents/RevealButton.java [hunk 2]
```diff
@@ -27,10 +25,7 @@ public RevealButton(GameController gameController, Skin skin) {
...
                 if (cell.containsFootstep()) {
                     gameController.actionHandler(Actions.REVEAL);
```
Comment (Joakim-Johansson) [inline Project/core/src/main/java/View/screen/GameScreenComponents/RevealButton.java:30]: This validation could probably be done in the actionHandler as actionHandler already retrieves the current cell which would remove some redundancy. I thinkMOVE does it similarly, where the handler is always run but it just runs break if it's invalid.

Comment (Vilmavds) [reply Project/core/src/main/java/View/screen/GameScreenComponents/RevealButton.java]: Updated now!

**Review by Joakim-Johansson — APPROVED** [review]

**Pull request merged**